README.md

```diff
@@ -13,3 +13,33 @@
   complexity suggests a different time.
 
 Add your answers to this markdown file.
+
+- Asymptotic analysis does not account for constants , which can be very misleading in the scope of comparing two algorithms. Foresay, if an algorithm with a better time complexity has a large constant, it could be much more time consuming for small inputs than an algorithm with a worse time complexity and much smaller constant.
+- Asmyptotic analysis focuses on the growth of input sizes (n) to determine how efficient an algorithm may be. This could result in a bad case with smaller inputs, where the speed may not reflect what the asymptotic analysis states since it focuses more so how it grows  in relation to large sizes rather than small sizes.
+- It does not account hardware in the analysis, which could be a huge factor being that is not reflected on.
+
+Assuming time complexity is `O(log(n))` , I would use log(10000) which is equal to about 13.3.
+
+Using this height of 13.3 we can divide it by 10 which is about the height for 1000 elements and multiply it by 5 seconds 
+
+13/10 * 5 = 6.65 seconds 
+
+- One reason could be a memory leak, where if the computer has ran out of memory it could start to make inefficient decisions to be make up for the leak. This could also be the result of a bad implementation that doesn't account for the memory efficiently. 
+- The binary tree could be inbalanced, which would then result in a huge time inefficiency because we are not having to traverse down each and every node for specific elements. This is a result of the tree being skewed heavily, where in the worst case it could start to look like a linked list. 
```

Comment:
So that would then be linear, i.e. you would expect it to take about 50 seconds, which is still much shorter than what it actually took.